permissions/plugin-azure-keyvault.yml

```diff
@@ -9,3 +9,4 @@ cd:
   enabled: true
 developers:
 - "timja"
+- "company-cloudbees-developers"
```

Comment:
? this isn't cloudbees maintained

Reply:
oops, yes - i mistakenly added the group to the wrong plugin, change reverted and correct plugin updated